build.gradle

```diff
@@ -2006,6 +2006,9 @@ allprojects {
     test {
         useJUnitPlatform();
 
+        // Always run tests
+        outputs.upToDateWhen { false }
+
```

Comment:
Should the 'testing' section of
https://wiki.openjdk.org/display/OpenJFX/Building+OpenJFX

be updated after this PR is integrated?

Are there any other wiki pages that might be impacted?

Reply:
Updating the testing section is a good idea. Thanks. I'll do that once I integrate.

I am not aware of any other page that needs to be updated.